t/RPC/multiple-rpcs.t

```diff
@@ -42,14 +42,23 @@ subtest 'RPCs should not block each others in the same service'  => sub {
 
         # if one RPC doesn't have messages it should not block the others
         for my $i (0..10) {
-            await (fmap {
+            await fmap_void(async sub {
                 my $rpc = shift;
-                $myriad->rpc_client->call_rpc('service.rpc', $rpc)->catch(sub {warn shift});
-            } foreach => ['echo', 'ping'], concurrent => 3);
+                my $response = await Future->needs_any(
+                    $myriad->rpc_client->call_rpc('service.rpc', $rpc)->catch(sub {warn shift}),
+                    # There is a timeout in place inside call_rpc, there is no need for this in real implementation.
+                    $myriad->loop->timeout_future(after => 1)
```

Comment:
I have a feeling that the timeout should be on all request, i.e all the request in less than a second, the current says every request can take a second, i.e `->timeout_future` should surround `fmap_void`.

Reply:
hmm yeah actually sure, then I can remove the check down there and only keep, that timeout one